app/views/student/profile.php

```diff
@@ -0,0 +1,43 @@
+<style>
+  .studentpic {
+    width: 200px;
+    height: 200px;
+    background: transparent no-repeat center center;
+    background-size: cover;
+    border-radius: 100px;
+    display: inline-block;
+  }
+  .student headline {
+    margin-bottom: -20px;
+  }
+  .student subheadline {
+    color: #000;
+  }
+  .studentinfo {
+    margin: 40px 0;
+  }
+  .topeditprofile {
+    margin-top: 20px;
+  }
+</style>
+
+<panel class="student">
+  <div class="content">
+    <headline><?php vecho('name'); ?></headline>
```

Comment:
Let's make sure to use the new View class, like View::echof and View::get instead of vecho and vget